src/neuroconv/tools/hdmf.py

```diff
@@ -1,11 +1,17 @@
 """Collection of modifications of HDMF functions that are to be tested/used on this repo until propagation upstream."""
 
 import math
+import warnings
 from typing import Tuple
 
 import numpy as np
 from hdmf.data_utils import GenericDataChunkIterator as HDMFGenericDataChunkIterator
 
+# Elevate any overflow warnings to trigger error, hopefully get full traceback too...
```

Comment:
Shouldn't this be inside the class at it is `__init__`. I am afraid this will modify a very general behavior from a very implicit / hidden place.

Can we constrain this to work only in the context of the changes here somehow?

Reply:
The `__init__` might not be the right place

Moved it inside the relevant local scopes though